src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -1213,6 +1239,17 @@ class HaAutomationPicker extends SubscribeMixin(LitElement) {
     }
   };
 
+  private _fetchDescriptions() {
+    this.automations.forEach((automation) =>
+      setEntityDescription(
+        this.hass,
+        automation.entity_id,
+        this._fetchedDescriptions,
+        getAutomationStateConfig
+      )
+    );
```

Comment:
This would fire a websocket request for each automation. Don't know if core supports it but we should batch this or just add the data in the original fetch.

Reply:
Is there a list of possible requests somewhere?